check-pr-commits.sh

```diff
@@ -173,7 +173,8 @@ check_commit() {
        is_merge_commit $sha || \
        subject_length_exceeds 70 $sha || \
        body_is_empty $sha || \
-       body_line_length_exceeds 78 $sha; then
+       body_line_length_exceeds 78 $sha || \
+       spellcheck $sha; then
```

Comment:
Should `spellcheck` be `typos`?

Reply:
Oh, I appear to have not committed my `spellcheck` function. 🤦